main.go

```diff
@@ -163,11 +164,18 @@ func buildAddonsRuntime(kubeC *rest.Config, mainFile string) (runtime.Runtime, e
 	return addons, nil
 }
 
+type verboseGlogWriter struct{}
+
+func (w *verboseGlogWriter) Write(p []byte) (n int, err error) {
+	log.V(1).Info(string(p))
```

Comment:
👍 one problem with just piping this into log.Info is p will have all the crap from `stdlog` like golog does over here https://github.com/golang/glog/blob/23def4e6c14b4da8ac2ed8007337bc5eb5007998/glog.go#L938

Reply:
so the output looks like 
```
I0319 21:47:49.872131   52277 main.go:170] 2020/03/19 21:47:49 proto: don't know how to copy 0
```

I just changed the stdlog format to remove the date and time and add the short filename

```
I0319 22:08:11.815655   53542 main.go:171] clone.go:234: proto: don't know how to copy 0
```
which is not too bad IMO since the log only shows in verbose mode, and more information may help with troubleshooting. 

What glog does internally is even better, but glog does not expose its `printWithFileLine` utility or the verbose level parsed from the command line. So essentially it requires reimplementing glog in isopod to make the output perfect, and it seems the gain does not outweigh the effort.